streaming_providers/debrid_client.py

```
@@ -1,121 +1,127 @@
-    def _parse_response(response, is_return_none):
+    async def _parse_response(response: httpx.Response, is_return_none: bool):
         if is_return_none:
             return {}
         try:
             return response.json()
-        except JSONDecodeError as error:
+        except ValueError as error:
             raise ProviderException(
                 f"Failed to parse response error: {error}. \nresponse: {response.text}",
                 "api_error.mp4",
             )
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)